front-end/src/components/Post/PostGovernanceInfo/PostReferendumInfo.tsx

```diff
@@ -39,12 +43,22 @@ const PostReferendumInfo = ({ onchainLink }: Props) => {
 				{(delay || delay === 0) &&
 					<Grid.Column mobile={16} tablet={8} computer={8}>
 						<h6>Delay</h6>
-						{delay}
+						<BlocksToTime blocks={delay} />
 					</Grid.Column>}
 				{end &&
 					<Grid.Column mobile={16} tablet={8} computer={8}>
-						<h6>End</h6>
-						{end}
+						{status === 'Started'
+							?
+							<>
+								<h6>Ends In</h6>
```

Comment:
I'll fight these one after the other, but this "all first caps" is not the default on Polkassembly.
```suggestion
								<h6>Ends in</h6>
```